### PR TITLE
Excel Options Component

### PR DESCRIPTION
**Pull request by nikitasheremet**
Build Excel Options component as per design: https://wolfbeacon.atlassian.net/wiki/spaces/CW/pages/1018789892/Excel+Download+Options+Component
Had to edit some logic in Date Picker component to allow for Datepicker to render without a default value.

### CODE REVIEW

File: src/Inputs/ExcelOptions.tsx [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+                >
+                    {keyToHeader(headersInSelect).map((header, index) => (
+                        <option key={header} value={header}>
+                            {header}
+                        </option>
+                    ))}
+                </Select>
+            </div>
+            <div>
+                <p>Date Picker</p>
```
Comment (ralph-dev) [inline src/Inputs/ExcelOptions.tsx]: CONSTANTS for From and To

File: src/Containers/Tag.tsx [hunk 2]
```diff
@@ -6,15 +6,17 @@ import { Main } from '@Utils/BaseStyles';
 
 export interface TagProps {
     icon?: React.ForwardRefExoticComponent<React.RefAttributes<SVGSVGElement>>;
+    onClick?: () => void;
```
Comment (fpunny) [inline src/Containers/Tag.tsx]: I know this is the way I do it, but look into extending TagProps with HTMLDivElement props, that would be better

File: src/Inputs/Datepicker/Datebox.tsx [hunk 3]
```diff
@@ -13,8 +13,11 @@ import { WEEKDAYS, MONTHS } from '@Utils/Constants';
 import { Button } from '../Button';
 
 const SIZE = 40;
-const displayDate = (date: Date): string =>
-    `${MONTHS[date.getMonth()]}, ${date.getFullYear()}`;
+
+const displayDate = (date: Date | undefined) =>
+    `${MONTHS[date ? date.getMonth() : new Date().getMonth()]}, ${
```
Comment (fpunny) [inline src/Inputs/Datepicker/Datebox.tsx]: Why not just make date have a default value of new Date?

File: src/Inputs/Datepicker/Datebox.tsx [hunk 4]
```diff
@@ -64,14 +67,16 @@ const buildCalendar = (
 
 export interface DateboxProps {
     changePage: (change?: number) => React.MouseEventHandler;
-    selectedDate: Date;
+    selectedDate: Date | undefined;
```
Comment (fpunny) [inline src/Inputs/Datepicker/Datebox.tsx]: ```selectedDate?: Date;```

**Pull request merged**